src/tactic/with_local_reducibility.lean

```diff
@@ -0,0 +1,46 @@
+import tactic.core
+
+namespace tactic
+
+/-- Possible reducibility attributes for a declaration:
+reducible, semireducible (the default), irreducible. -/
+@[derive decidable_eq]
+inductive decl_reducibility
+| reducible
+| semireducible
+| irreducible
+
```

Comment:
```suggestion

instance : inhabited decl_reducibility := ⟨decl_reducibility.semireducible⟩

```

Reply:
This would satisfy the inhabited linter.

Reply:
Or maybe `@[derive inhabited]`?

Reply:
that would make the default `reducible`. I don't know if changing the order of constructors here would break other things...

Reply:
Oh, good point.

Reply:
Changing the order wouldn't break anything, but explicit is better than implicit IMHO. (I'm not sure the inhabited linter really makes sense, especially for tactic stuff, but I've just added an instance to make it happy.)